src/custom/components/Tooltip/TooltipMod.tsx

```diff
@@ -16,14 +16,18 @@ export interface TooltipProps extends Omit<PopoverProps, 'content' | 'PopoverCon
 
 interface TooltipContentProps extends Omit<PopoverProps, 'content' | 'PopoverContainer' | 'Arrow'> {
   content: ReactNode
+  onOpen?: () => void
+  // whether to wrap the content in a `TooltipContainer`
+  // wrap?: boolean
```

Comment:
Maybe we could keep this option and set the default to `true` which is the current behaviour